### ISSUE TITLE
Can't use std::borrow::Cow<'static, str> as label value

### ISSUE THREAD

**Issue by RAnders00** [metrics-rs/metrics#162]
Example that doesn't work:

```rust
use std::borrow::Cow;
use metrics;

fn main() {
    let label_value: Cow<'static, str> = Cow::Borrowed("def");

    metrics::gauge!("test", 123.0, "asd" => label_value.clone());
}
```

Error:

```
error[E0277]: the trait bound `metrics::cow::Cow<'static, str>: From<Cow<'_, str>>` is not satisfied
  --> src/main.rs:7:5
   |
7  |     metrics::gauge!("test", 123.0, "asd" => label_value.clone());
   |     ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^ the trait `From<Cow<'_, str>>` is not implemented for `metrics::cow::Cow<'static, str>`
   |
  ::: /home/ruben/.cargo/registry/src/github.com-1ecc6299db9ec823/metrics-0.13.0/src/label.rs:22:12
   |
22 |         V: Into<SharedString>,
   |            ------------------ required by this bound in `Label::new`
   |
   = help: the following implementations were found:
             <metrics::cow::Cow<'_, [Label]> as From<Vec<Label>>>
             <metrics::cow::Cow<'_, [metrics::cow::Cow<'static, str>]> as From<Vec<metrics::cow::Cow<'static, str>>>>
             <metrics::cow::Cow<'_, str> as From<String>>
             <metrics::cow::Cow<'a, T> as From<&'a T>>
   = note: required because of the requirements on the impl of `Into<metrics::cow::Cow<'static, str>>` for `Cow<'_, str>`
   = note: this error originates in a macro (in Nightly builds, run with -Z macro-backtrace for more info)
```

I would expect there to be a `impl From<std::borrow::Cow> for metrics::cow::Cow`

**Comment by tobz** [comment]
At face value, this seems reasonable/doable.

Happy to accept a PR, otherwise it may take some time longer.

**Comment by tobz** [comment]
Addressed this in f4c0555 and released `metrics@v0.13.1`.
